### PR TITLE
feature/742/fix-view-sorry-block

### PR DESCRIPTION
**Pull request by ArturBekhDEV**
updated view sorry block and also add scroll to filter menu when 'sorry-block' appears.
pull1
pull2

### CODE REVIEW

File: src/pages/find-offers/FindOffers.tsx [hunk 1]
```diff
@@ -117,6 +117,17 @@ const FindOffers = () => {
     filterQueryActions.updateFilterInQuery(updatedRole, 'authorRole')
   }
 
+  const getFilterSectionStyles = useMemo(() => {
+    return !offers.length && isDesktop
+      ? {
+          maxHeight: '500px',
```
Comment (MickhaelSh) [inline src/pages/find-offers/FindOffers.tsx:123]: styles to styles file

**Review by maarekua — APPROVED** [review]

File: src/pages/find-offers/FindOffers.tsx (hunk 1 above)
Comment (ArturBekhDEV) [reply src/pages/find-offers/FindOffers.tsx]: fixed

File: src/pages/find-offers/FindOffers.tsx [hunk 2]
```diff
@@ -168,14 +189,27 @@ const FindOffers = () => {
...
-            isFiltersOpen={isOpen}
-            offerCards={offers}
-            viewMode={cardsView}
-          />
+          <Box
+            onLoad={updateOfferContainerHeight}
+            sx={styles.offerContainerHeight}
+          >
+            {!offers.length && !offersLoading ? (
+              <Box ref={offerContainerRef}>
```
Comment (Yurenko) [inline src/pages/find-offers/FindOffers.tsx:198]: You don't need to add two boxes with the same ref.
Do it like this:
<Box
            onLoad={updateOfferContainerHeight}
            sx={styles.offerContainerHeight}
            ref={offerContainerRef}
          >

File: src/pages/find-offers/FindOffers.styles.ts [hunk 3]
```diff
@@ -23,5 +23,17 @@ export const styles = {
...
+    paddingLeft: '30px'
+  },
+  filterSectionNotFound: (height: number) => ({
+    maxHeight: `${height}px`,
+    overflowY: 'auto'
+  }),
+  filterSectionStyles: (isDesktop: boolean, height: number) => ({
+    maxHeight: !isDesktop ? '100%' : `${height}px`,
+    overflowY: !isDesktop ? 'visible' : 'auto'
+  })
```
Comment (Yurenko) [inline src/pages/find-offers/FindOffers.styles.ts:38]: You can use breakpoints for styles

**Review by maarekua — APPROVED** [review]

**Review by mxrcury — APPROVED** [review]

**Pull request closed without merging**